src/main/groovy/com/ullink/gradle/nunit/ReportGenerator.groovy

```diff
@@ -0,0 +1,116 @@
+package com.ullink.gradle.nunit
+
+import org.gradle.api.tasks.Exec
+
+class ReportGenerator extends Exec {
```

Comment:
it seems it worth to be a separate gradle plugin?

Reply:
Will move the entire code to https://github.com/Ullink/gradle-extentreports-dotnet-plugin and do the followups there.